src/vulnerability-report.ts

```diff
@@ -9,7 +19,15 @@ export class Vulnerability {
     severity: string;
     links?: string[];
 
-    constructor(params: Vulnerability) {
+    //
```

Comment:
You can leave such comment as PR comment, but it's not useful as part of the source code.